driver/api/impl/impl.cpp

```diff
@@ -659,6 +730,34 @@ SQLRETURN BindParameter(
         const auto apd_record_count = apd_desc.getRecordCount();
         const auto ipd_record_count = ipd_desc.getRecordCount();
 
+        // If parameter_value_ptr and StrLen_or_IndPtr pointers are both NULL, then unbind the parameter,
+        // and if the parameter number is [greater than or] equal to the highest bound parameter number,
+        // then adjust SQL_DESC_COUNT to reflect that the highest bound parameter has chaged.
+        if (
+            parameter_value_ptr == nullptr &&
+            StrLen_or_IndPtr == nullptr &&
+            parameter_number >= apd_record_count
+        ) {
+            auto nextHighestBoundParameterNumber = std::min<std::size_t>(apd_record_count, parameter_number - 1);
+
+            while (nextHighestBoundParameterNumber > 0) {
+                auto & apd_record = apd_desc.getRecord(nextHighestBoundParameterNumber, SQL_ATTR_APP_PARAM_DESC);
+
+                if (
+                    apd_record.getAttrAs<SQLPOINTER>(SQL_DESC_DATA_PTR, 0) != nullptr ||
```

Comment:
Finding the latest bound parameter (one with non-null data ptr or length ptr, or indicator ptr)

Reply:
Is you message truncated?

Reply:
sorry, that was an intermediate note for myself